### DISCUSSION TITLE
Disabling the black screen in unconscious state

### DISCUSSION THREAD

**Discussion by alezm00** [acemod/ACE3#9315] [Q&A] [answered]
Hi i'm looking into disabling the black screen when the player is in unconscious state but I want to leave the blur...

i was thinking of deleting the `ppUnconsciousBlackout` effect but it would be a workaround...

thx <3

**Comment by acknowledge14** [comment, marked as answer]
Maybe you can check this out: ACE Arcade - Configureable Simplified Mechanics
https://steamcommunity.com/sharedfiles/filedetails/?id=2947221921&searchtext=ace+arcade

**Reply by alezm00** [reply]
found my answer just delete the effect thanks
